src/pretix/plugins/banktransfer/payment.py

```diff
@@ -118,6 +118,16 @@ def test_mode_message(self):
     def settings_form_fields(self):
         d = OrderedDict(
             list(super().settings_form_fields.items()) + list(BankTransfer.form_fields().items()) + [
+                ('code_prefix', forms.CharField(
+                    label=_('Bank transfer code prefix.'),
+                    help_text=_('This value is prepended to the unique order code.'),
+                    widget=forms.TextInput(
+                        attrs={
+                            'placeholder': self.event.slug.upper(),
+                        },
+                    ),
+                    required=False
```

Comment:
We might want to add a validator to make sure only letters and characters are used. Everything else will both cause problems with the banks and with the RegEx generation.